src/components/ProgressForm/ProgressForm.module.scss

```diff
@@ -0,0 +1,77 @@
+$base-unit: 4px;
+$color-blue: #041187;
+$color-red: #e30062;
+$color-green: #85da6b;
+$color-disabled: #e5e7eb;
+$diff-margin: $base-unit * 15;
+$similar-margin: $base-unit * 8;
+$section-padding-top-bottom: $base-unit * 30;
+$section-padding-left-right: $base-unit * 25;
+
+.header {
+    margin: $diff-margin auto;
+    background-color: $color-blue;
+    color: white;
```

Comment:
use color variable

Reply:
I already have it as a variable

Reply:
Can you please point me to where you have defined it as a variable

Reply:
sorry, my bad. Resolved them